client/dashboard/sites/overview/uptime-card.tsx

```diff
@@ -17,18 +37,33 @@ export default function UptimeCard( { siteMonitorUptime }: { siteMonitorUptime?:
 	);
 	const uptimePercentage = Math.round( ( ( upDays / ( upDays + downDays ) ) * 1000 ) / 10 );
 	return (
-		<>
-			<OverviewCard
-				title={ __( 'Uptime' ) }
-				icon={ connection }
-				heading={
-					/* translators: %s: percentage of site uptime. Eg. 99% */
-					sprintf( __( '%s%%' ), uptimePercentage )
-				}
-				metaText={ __( 'Past 30 days' ) }
-			>
-				<OverviewCardProgressBar value={ uptimePercentage } />
-			</OverviewCard>
-		</>
+		<OverviewCard
+			title={ __( 'Uptime' ) }
+			icon={ connection }
+			heading={
+				/* translators: %s: percentage of site uptime. Eg. 99% */
+				sprintf( __( '%s%%' ), uptimePercentage )
+			}
+			metaText={ __( 'Past 30 days' ) }
+		>
+			<OverviewCardProgressBar value={ uptimePercentage } />
+		</OverviewCard>
+	);
+}
+
+function UptimeCardDisabled() {
+	return (
+		<OverviewCard title={ __( 'Uptime' ) } icon={ connection } heading={ __( 'Unknown' ) }>
+			<OverviewCardProgressBar value={ 0 } />
+			<ExternalLink href="#">{ __( 'Get Jetpack Monitor' ) }</ExternalLink>
+		</OverviewCard>
+	);
+}
+
+export default function UptimeCard( { site }: { site: Site } ) {
+	return site.jetpack && site.jetpack_modules.includes( 'monitor' ) ? (
```

Comment:
Is this condition correct? I can't see it when using any of my WoA sites.

In any case, is this necessary in this PR? I haven't heard anything from the designers regarding upsells in this screen 🤔

Reply:
Can you try with a wpcom Atomic site?

Reply:
> I haven't heard anything from the designers regarding upsells in this screen

The whole screen is TBD, so I don't think it matters either way. I can propose keeping this upsell when the design is finalised.